tests/sentry/buffer/test_redis.py

```diff
@@ -342,6 +342,14 @@ def test_buffer_hook_registry(self):
         assert mock.call_count == 1
         assert mock.call_args[0][0] == self.buf
 
+    def test_process_batch(self):
```

Comment:
would be nice to include a test to validate that the partitioning code is working as expected in process_pending.

Reply:
for posterity (discussed in Slack) this is tested by https://github.com/getsentry/sentry/blob/master/tests/sentry/buffer/test_redis.py#L206